packages/ckeditor5-list/tests/todolist/todolistediting.js

```diff
@@ -1265,6 +1265,8 @@ describe( 'TodoListEditing - checkbox rendering', () => {
 
 		expect( checkboxElement.tagName ).to.equal( 'INPUT' );
 		expect( checkboxElement.checked ).to.equal( false );
+		expect( checkboxElement.hasAttribute( 'tabindex' ) ).to.be.true;
```

Comment:
This assertion is obsolete. `Element#getAttribute()` always returns `null` for a non-existing name. The next assertion will handle this.